guides/common/modules/proc_enabling-client-connections-to-satellite.adoc

```diff
@@ -14,18 +14,20 @@ include::snip_firewalld.adoc[]
 [options="nowrap"]
 ----
 # firewall-cmd \
---add-port="53/udp" --add-port="53/tcp" \
---add-port="67/udp" \
---add-port="69/udp" \
---add-port="80/tcp" --add-port="443/tcp" \
+--add-service=dns \
+--add-service=dhcp \
+--add-service=tftp \
```

Comment:
Some discussion: we don't enable dns, dhcp and tftp by default, so why do we open the ports by default? Shouldn't it be part of the chapters where we do enable the services?

Reply:
I we are able to reliably track all the procedures which require enabling these services then adding them to firewall rules can certainly be moved there. That being said, I don't know how many of these procedures are there.

Reply:
DNS: `--foreman-proxy-dns true --foreman-proxy-dns-managed true` then we install a DNS server and DNS needs to be opened
DHCP: `--foreman-proxy-dhcp true --foreman-proxy-dhcp-managed true`
TFTP: `--foreman-proxy-tftp true --foreman-proxy-tftp-managed true`

Note we often `--foreman-proxy-dhcp=true` instead of `--foreman-proxy-dhcp true` in docs, so careful when grepping. Sometimes we omit the `-managed true` part since that's default, but technically we should include it when we rely on it to be sure.